src/geometry/Shape.ts

```diff
@@ -257,7 +266,7 @@ export default abstract class Shape extends THREE.Group {
     };
   }
 
-  restyle(style: Style): void {
+  restyle(style: Style, config: { includeDescendents: boolean } = { includeDescendents: false }): void {
```

Comment:
Make includeDescendants a parameter passed together with the Style, i.e. change the type to `Style & { includeDescendants: boolean }`. Also double check the spelling